kernel_gateway/proposal.md

```diff
@@ -0,0 +1,90 @@
+# Proposal for Incubation
+
+## Subproject name
+
+Jupyter Kernel Gateway (`jupyter-incubator/kernel_gateway`)
+
+## Development team and Advocate
+
+* Peter Parente, IBM (`@parente`)
+* Dan Gisolfi, IBM (`@vinomaster`)
+* Justin Tyberg, IBM (`@jtyberg`)
+* Gino Bustelo, IBM (`@ginobustelo`)
+
+Who is the Steering Council Advocate for this Subproject?
+
+* Kyle Kelley (`@rgbkrk`)
+
+## Subproject goals, scope and functionality
+
+### Problem
+
+Applications that use Jupyter kernels as execution engines outside of the traditional notebook / console user experience have started appearing on the web (e.g., [pyxie](https://github.com/oreillymedia/pyxie-static), [pyxie kernel server](https://github.com/oreillymedia/ipython-kernel), [Thebe](https://github.com/oreillymedia/thebe), [gist exec](https://github.com/rgbkrk/gistexec), [notebooks-to-dashboards](http://blog.ibmjstart.net/2015/08/22/dynamic-dashboards-from-jupyter-notebooks/), ). Today, these projects rely on:
```

Comment:
While I am +100 on these ideas, I don't think they are only userful for non-notebook clients. Have you thought about the aspects of this that integrate with the actual notebook and jupyterhub?